sources/packages/api/src/database/entities/institution-user-auth.model.ts

```diff
@@ -0,0 +1,38 @@
+import { Entity, JoinColumn, ManyToOne, PrimaryGeneratedColumn } from "typeorm";
+import { InstitutionLocation } from "./institution-location.model";
+import { InstitutionUserTypeAndRole } from "./institution-user-type-role.model";
+import { InstitutionUser } from "./institution-user.model";
+import { RecordDataModel } from "./record.model";
+
+@Entity({
+  name: "institution_user_auth",
+})
+export class InstitutionUserAuth extends RecordDataModel {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @ManyToOne((type) => InstitutionLocation, { eager: true })
+  @JoinColumn({
+    name: "institution_location_id",
+    referencedColumnName: "id",
+  })
+  location: InstitutionLocation;
+
+  @ManyToOne((type) => InstitutionUserTypeAndRole, { eager: true })
```

Comment:
I would recommend removing the eager loading and letting the developer decide when it makes sense to retrieve the relational data and which columns are needed.

Reply:
I think eager loading for location is not required but TypeAndRole should be eager .... any development perspective I don't see any case where we need that entity without that type.